src/modules/Builder/features/ActivityItems/LeftBar/Item/Item.styles.ts

```diff
@@ -40,15 +41,25 @@ export const StyledItem = styled(StyledFlexTopCenter)`
     ${commonButtonStyles};
   }
 
+  ${({ isActive }: { isActive: boolean }) =>
```

Comment:
pls add hasError: boolean to typescript types and remove on bottom:
${({ isActive }: { isActive: boolean, hasErrror: boolean, hidden?: boolean }) => ...
${({ hasError }) => ...
${({ isActive, hasError }) => ...
and so on

Reply:
done